src/os/bluestore/BlueStore.h

```diff
@@ -2137,7 +2137,12 @@ class BlueStore : public ObjectStore,
 		std::numeric_limits<decltype(min_alloc_size)>::digits,
 		"not enough bits for min_alloc_size");
 
-  bool per_pool_omap = false;
+  enum {
+    // Please preserve the order since it's DB persistent
+    OMAP_UNSORTED = 0,
```

Comment:
How about changing "OMAP_UNSORTED" to "OMAP_BULK"?
It is just confusing because it is obviously sorted anyway.